pkgs/fast_barcode_scanner/lib/src/fast_barcode_scanner.dart

```diff
@@ -78,6 +81,19 @@ class FastBarcodeScannerController implements BarcodeScannerController {
   void start() {
     fbs.CameraController.instance.resumeDetector();
   }
+
+  @override
+  Future<bool> get supportsSwitchingCamera =>
+      fbs.CameraController.instance.canChangeCamera();
+
+  @override
+  Future<void> toggleCamera() => fbs.CameraController.instance.toggleCamera();
+
+  @override
+  Future<void> toggleTorch() => fbs.CameraController.instance.toggleTorch();
+
+  @override
+  bool get torchState => fbs.CameraController.instance.state.torchState;
```

Comment:
what does torchState mean? `torchState` is a bad name chosen by the package we are using. we should make it better